app/models/meals/meal.rb

```diff
@@ -33,7 +33,7 @@ class Meal < ApplicationRecord
     has_many :reminder_deliveries, class_name: "Meals::RoleReminderDelivery", inverse_of: :meal,
       dependent: :destroy
     has_many :transactions, class_name: "Billing::Transaction", as: :statementable,
-      dependent: :restrict_with_exception, inverse_of: :statementable
+      dependent: :destroy, inverse_of: :statementable
```

Comment:
This is the only dependent: :destroy change that may be an issue. I'm guessing this is restricted since transactions should never be destroyed in production?

Reply:
It should be fine since meals aren't allowed to be destroyed if they are finalized. The only way we'd have an issue if there was a race condition in which a meal got destroyed right as it was being finalized. Very unlikely.